modules/compression/src/lib/zstd-compression.ts

```diff
@@ -50,4 +52,18 @@ export class ZstdCompression extends Compression {
     const inputArray = new Uint8Array(input);
     return simpleZstd.decompress(inputArray).buffer;
   }
+
+  async decompress(input: ArrayBuffer, size?: number): Promise<ArrayBuffer> {
```

Comment:
Nice. To really make this fit with the loaders.gl Compression model
- Can we implement `ZstdCompression.decompressInBatches()` similar to `DeflateCompression`?
- And then call this.decompressInBatches() inside this function?

Reply:
This is going to be an external chunks support with combination of sync decompression. I'd prefer built-in chunks support of `zstd-codec`

Reply:
Well, any reason we can't we support both? 
- Basically big files should not have to be assembled into a big buffer and then split up again - which is what this function requires.
- The compressor should be able to work on chunks directly from the input stream without requiring all data to be loaded in advance.

Reply:
I mean we can have `ZstdCompression.decompressInBatches()` but we won't use it in `decompress` because we prefer built-in `decompressChunks`